### PR TITLE
Bugfix/atr 274 dev

### PR DESCRIPTION
**Pull request by SENya1990**

### CODE REVIEW

**Comment by dropsonic** [comment]
@SENya1990 there is a mix of tabs and spaces all over the pull request. It makes it almost impossible to read.

**Comment by SENya1990** [comment]
@dropsonic I've tried to fix tabs. One more try

File: src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/ITypeSymbolExtensions.cs [hunk 1]
```diff
@@ -331,27 +341,29 @@ private static string GetClrStyleTypeFullNameForNotNestedType(this ITypeSymbol n
             return initializers;
         }
 
-        public static ImmutableArray<StaticConstructorInfo> GetStaticConstructors
-            (this INamedTypeSymbol typeSymbol, CancellationToken cancellation = default)
-        {
-            typeSymbol.ThrowOnNull(nameof(typeSymbol));
+		public static ImmutableArray<StaticConstructorInfo> GetStaticConstructors
```
Comment (dropsonic) [inline src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/ITypeSymbolExtensions.cs]: tabs/spaces mix

Comment (SENya1990) [reply src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/ITypeSymbolExtensions.cs]: The VS shows no mixed tabs. I think the method was created with this indentation. I'll fix it

Comment (SENya1990) [reply src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/ITypeSymbolExtensions.cs]: fixed

**Comment by SENya1990** [comment]
@dropsonic isn't it our code style formatting is set to use Keep tabs option and Smart Indent? What's the problem with spaces replaced to tabs? It's standard VS formatting for these options

**Pull request merged**